public/js/clients/firefox.js

```diff
@@ -101,9 +101,8 @@ function initPage(actions) {
   threadClient = getThreadClient();
 
   setupCommands({ threadClient, tabTarget, debuggerClient });
-
   tabTarget.on("will-navigate", actions.willNavigate);
-  tabTarget.on("navigate", actions.navigate);
+  tabTarget.on("navigate", actions.willNavigate);
```

Comment:
I think is probably better to make the `navigate` action meaningful than to change to `willNavigate`